vspec2csv.py

```diff
@@ -40,13 +40,8 @@ def print_csv_content(file, tree):
         data_type_str = tree_node.data_type.value if tree_node.has_data_type() else ""
         unit_str = tree_node.unit.value if tree_node.has_unit() else ""
 
-        if tree_node.instances:
```

Comment:
Tree is flattened before we write *.csv so there will never be any instances. I verified that we generate the same number of lines after this change as before

Reply:
I like this change but would have preferred it in a separate commit.

Reply:
Will put the csv part in a separate commit